package.json

```diff
@@ -22,16 +22,16 @@
   },
   "homepage": "https://github.com/ember-cli/ember-cli-preprocessor-registry#readme",
   "dependencies": {
-    "broccoli-clean-css": "^1.1.0",
-    "broccoli-funnel": "^1.0.0",
-    "broccoli-merge-trees": "^1.0.0",
-    "debug": "^2.2.0",
-    "ember-cli-lodash-subset": "^1.0.7",
+    "broccoli-clean-css": "^2.0.1",
+    "broccoli-funnel": "^2.0.1",
+    "broccoli-merge-trees": "^3.0.1",
+    "debug": "^3.0.1",
+    "ember-cli-lodash-subset": "^2.0.1",
```

Comment:
do we need `ember-cli-lodash-subset` at all?

Reply:
Yes, it's used in two other places (`find` and `uniq`)

Reply:
find is a native function now, isn't it? (at least on arrays?)

Reply:
Yes, I was thinking about replacing it, although `uniq` is still needed.